update-submodule/action.yml

```diff
@@ -82,6 +89,7 @@ runs:
         fetch-depth: 0
 
     - name: Create the new branch and push changes
+      id: create-branch
```

Comment:
Shouldn't be it `submodule-update`? `create-branch` sounds weird.

Reply:
Although, I am okay. Please ignore.

Reply:
there are two steps: create branch, create pr.